apps/api/Dockerfile

```
@@ -41,14 +31,12 @@ RUN cd /app/sharedLibs/html-transformer && \
     chmod +x target/release/libhtml_transformer.so
 
 FROM base
-COPY --from=prod-deps /app/node_modules /app/node_modules
 COPY --from=build /app /app
```

Not convinced this is the right move here, as we specifically only want the prod deps to be in the resulting image. A better solution seems to me like re-adding the prod-deps build and moving the copy order around.
```suggestion
COPY --from=build /app /app
COPY --from=prod-deps /app/node_modules /app/node_modules
```

---

Yes I agree, I think you should copy /app/dist from the build docker . not /app
Cf https://pnpm.io/docker

---

I've just tried and it doesn't run. It does compile but then 
```
Cannot find module 'mammoth'
Cannot find module 'express'
```